keystone/contrib/oauth2/backends/sql.py

```diff
@@ -17,9 +17,10 @@
 from keystone.common import sql
 from keystone.contrib import oauth2
 from keystone import exception
-from keystone.i18n import _
-from oslo.utils import timeutils
-
+#from keystone.i18n import _
+from keystone.openstack.common.gettextutils import _
+#from oslo.utils import timeutils
+from keystone.openstack.common import timeutils
```

Comment:
This line makes the test fail with a `ImportError: cannot import name timeutils`